qadence/analog/utils.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import annotations
+
+from math import dist as euclidean_distance
+
+from sympy import cos, sin
+
+from qadence.blocks.abstract import AbstractBlock
+from qadence.blocks.analog import (
+    ConstantAnalogRotation,
+)
+from qadence.operations import N, X, Y, add, kron
+from qadence.register import Register
+
+C6_DICT = {
```

Comment:
Maybe add a comment about what this dict is all about ? Or a more meaningful name ? It is Rydberg level to coupling strength I guess. Also it would be worth mentioning the units I think.

Reply:
Yes indeed